ch.bailu.gtk_meteo.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app-id": "ch.bailu.gtk_meteo",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "gtk-meteo.sh",
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk17/install.sh"
+      ]
+    },
+    {
+      "name": "gtk-meteo",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/bailuk/gtk-meteo.git",
+          "branch": "543eb0a5ff40b57d70277fb6c7ac9b3e6f9fb431"
+        },
+        "gradle-sources.json"
+      ],
+      "build-commands": [
+        "sed -i s/distributionUrl.*/distributionUrl=gradle-bin.zip/ gradle/wrapper/gradle-wrapper.properties",
+        "source /usr/lib/sdk/openjdk17/enable.sh && ./gradlew build",
+        "install -Dm755 ci/flatpak-run-app.sh /app/bin/gtk-meteo.sh",
+        "install -Dm644 build/libs/gtk-meteo-all.jar /app/lib/gtk-meteo.jar",
+        "install -Dm644 flatpak/ch.bailu.gtk_meteo.desktop /app/share/applications/ch.bailu.gtk_meteo.desktop",
+        "install -Dm644 flatpak/ch.bailu.gtk_meteo.metainfo.xml /app/share/metainfo/ch.bailu.gtk_meteo.metainfo.xml",
+        "install -Dm644 src/main/resources/svg/app-icon.svg /app/share/icons/hicolor/scalable/apps/ch.bailu.gtk_meteo.svg"
+      ]
+    }
+  ],
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--env=GIO_EXTRA_MODULES=/app/lib/gio/modules",
+    "--env=JAVA_HOME=/app/jre",
+    "--env=LIBO_FLATPAK=1",
+    "--env=PATH=/app/bin:/app/jre/bin",
+    "--filesystem=~/.config/gtk-meteo",
+    "--filesystem=~/.java/.userPrefs/gtk-meteo"
```

Comment:
these two should be "persist" instead. 

also are you sure it's `.config` and not `$XDG_CONFIG_HOME` ?

Reply:
Upstream now uses $XDG_CONFIG_HOME